MAINTAINERS.md

```diff
@@ -1,238 +1,82 @@
-# Maintainers' Guide
+# Maintaining
 
-## Branch organization
+## Overview
 
-The essence of this branching scheme is that you create "release branches" of
-independently releasable units of work.  These can then be released by their
-maintainers when ready.
+We use a three-stage release pipeline. Each stage of the release pipeline has
+its own dedicated branch and corresponding testing moon. Features and bug fixes
+progress through each stage--and are subject to testing along the way--until
+they're eventually released to the live network. This pipeline automates our
+release process, making it much easier to quickly and reliably ship code. It's
+also simple to reason about.
 
-### Master branch
+## Branches and Moons
 
-Master is what's released on the network.  Deployment instructions are in the
-next section, but tagged releases should always come from this branch.
+The branches and their corresponding moons that comprise the stages of the
+release pipeline are:
 
-### Feature branches
+--------------------------------------------------------------
+ Branch     |            Moon          | Target audience
+--------------------------------------------------------------
+`develop`   | `~binnec-dozzod-marzod`  | Kernel developers  
+`release`   | `~marnec-dozzod-marzod`  | Early Adopters  
+`release`   | `~doznec-dozzod-marzod`  | App Developers  
```

Comment:
I'd think app developers should receive releases earlier in the cycle than early adopters. I'd consider them more likely to uncover issues since they interface directly with kernel APIs.